modules/claims_api/spec/fixtures/form_526_evss.json

```diff
@@ -1 +1 @@
-{"form526":{"veteran":{"currentlyVAEmployee":false,"currentMailingAddress":{"city":"Portland","country":"USA","zipFirstFive":"12345","addressLine1":"1234 Couch Street","addressLine2":"Apt. 22","type":"DOMESTIC","zipLastFour":"6789","state":"OR"}},"applicationExpirationDate":"2015-08-28T19:53:45+00:00","serviceInformation":{"servicePeriods":[{"activeDutyEndDate":"1990-01-02","serviceBranch":"Air Force","activeDutyBeginDate":"1980-02-05"},{"activeDutyEndDate":"1999-01-01","serviceBranch":"Air Force Reserve","activeDutyBeginDate":"1990-04-05"}]},"disabilities":[{"ratedDisabilityId":"1100583","diagnosticCode":9999,"disabilityActionType":"NEW","name":"PTSD (post traumatic stress disorder)","secondaryDisabilities":[{"name":"PTSD personal trauma","disabilityActionType":"SECONDARY","serviceRelevance":"Caused by a service-connected disability\\nLengthy description"}]}],"claimantCertification":true},"form526_uploads":[],"form4142":null,"form0781":null,"form8940":null}
\ No newline at end of file
+{"form526":{"veteran":{"currentlyVAEmployee":false,"currentMailingAddress":{"city":"Portland","country":"USA","zipFirstFive":"12345","addressLine1":"1234 Couch Street","addressLine2":"Apt. 22","type":"DOMESTIC","zipLastFour":"6789","state":"OR"}},"applicationExpirationDate":"2015-08-28T19:53:45+00:00","serviceInformation":{"servicePeriods":[{"activeDutyEndDate":"1990-01-02","serviceBranch":"Air Force","activeDutyBeginDate":"1980-02-05"},{"activeDutyEndDate":"1999-01-01","serviceBranch":"Air Force Reserve","activeDutyBeginDate":"1990-04-05"}]},"disabilities":[{"ratedDisabilityId":"1100583","diagnosticCode":9999,"disabilityActionType":"NEW","name":"PTSD (post traumatic stress disorder)","secondaryDisabilities":[{"name":"PTSD personal trauma","disabilityActionType":"SECONDARY","serviceRelevance":"Caused by a service-connected disability\\nLengthy description"}]}],"claimantCertification":true,"autoCestPDFGenerationDisabled":false},"form526_uploads":[],"form4142":null,"form0781":null,"form8940":null}
```

Comment:
Why is this one not pretty-formatted like the other fixtures? Makes the diff *really* hard to find.